docker-compose.yml-example

```diff
@@ -2,21 +2,20 @@ version: "3"
 
 services:
   web:
-    build: caddy/
-    #image: abiosoft/caddy:0.10.4
+    build: kweb/
+    image: ${docker_repo:?err}/kopano_web
     container_name: web
     restart: always
-    privileged: true
     links:
       - kopano_webapp
       - kopano_zpush
     ports:
       - "2015:2015"
       - "${HTTP}:80"
       - "${HTTPS}:443"
+    command: caddy -conf /etc/Caddyfile -agree -email ${EMAIL} -host ${FQDN}
```

Comment:
Rename /etc/Caddyfile to /etc/kweb.cfg or something.

Reply:
This needs checking if that start up fine when fqdn is given. Probably needs checks to avoid those parameters when it the variables are not given (needs a docker-run script).

Reply:
renamed and yes, starts up with these parameters (well at least if the env variables do exist)